docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
       - "6379:6379"
 
   avalon:
-    image: avalonmediasystem/avalon:6.x-dev
+    # image: avalonmediasystem/avalon:6.x-dev
```

Comment:
Is this an intended related change?

Reply:
Yes, upstream is no longer updating 6x Docker images and the tag locks NodeJS to v8 which is end-of-life and unsupported by some nodejs packages in this update.